src/main/java/appeng/tile/inventory/AppEngFilteredInvWrapper.java

```diff
@@ -0,0 +1,63 @@
+
+package appeng.tile.inventory;
+
+
+import javax.annotation.Nonnull;
+
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.items.IItemHandlerModifiable;
+
+
+public class AppEngFilteredInvWrapper implements IItemHandlerModifiable
+{
+	final private IItemHandlerModifiable handler;
+	final private IAEItemFilter filter;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AModifiersOrderCheck)